lib/rubygems/exceptions.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
-# TODO: the documentation in here is terrible.
-#
+
 # Each exception needs a brief description and the scenarios where it is
```

Comment:
:thinking: I think those two lines are explaining the original `TODO` problem and should be removed as well if `TODO` line is going to be removed.

Reply:
Thanks @simi, you are right  👍